opm/parser/eclipse/EclipseState/Schedule/MSW/SegmentSet.cpp

```diff
@@ -21,6 +21,11 @@
 #include <cmath>
 #include <map>
 
+#ifdef _WIN32
```

Comment:
No boost; this is not enough of a reason to increase boost usage. I thought the `static` should be sufficient to protect against possible naming conflicts, an alternative would be to put it in an anonymous namespace.